### ISSUE TITLE
Javacore 1CIJAVAVERSION does not detect Windows Server 2019

### ISSUE THREAD

**Issue by paulcheeseman** [eclipse-openj9/openj9#16126]
Java -version output
--------------------

Reported on IBM Java 8.0.6.20, but this looks like it will affect any JDK that uses OpenJ9/OMR.

Summary of problem
------------------

In javacores taken on Windows Server 2019, the Java version information shows the OS as Windows Server 2012, i.e.:

```
1CIJAVAVERSION JRE 1.8.0 Windows Server 2012 amd64-64 (build 8.0.6.20 - pwa6480sr6fp20-20201120_02(SR6 FP20))
```

I think this version information is populated by the following code, which appears to have no way of detecting Windows Server 2019:

https://github.com/eclipse/omr/blob/e82572ab6ee5ee0c78b9c9f9228e87a7fa958766/port/win32/omrsysinfo.c#L320-L351

**Comment by pshipton** [comment]
https://github.com/eclipse/omr/pull/3648 added support for detecting Windows Server 2019. We're looking specifically for build 1809, perhaps there are other builds of Server 2019. What does `ver` from a command prompt show?

https://github.com/eclipse/omr/pull/6259 from SR7 FP10 improves support to look for builds >= 17763 and < 20348.

Does FP25 detect Windows Server 2019 correctly? The original change is first in SR6 FP20, but seems to me the java launcher also needs to declare support for Server 2019, otherwise the app runs in a compatibility mode that looks like the previous version.

**Comment by pshipton** [comment]
I tried a handy Server 2019 server machines and it shows build 17763. I expect you need 8.0.7.10 to fix the OS detection.

**Comment by pshipton** [comment]
Actually I guess 1809 isn't a build number but the ReleaseId. You can look in the registry, at ReleaseId under SOFTWARE\\Microsoft\\Windows NT\\CurrentVersion. Or use PowerShell and run `Get-ComputerInfo -Property "WindowsVersion"`.

On the machine I'm using the javacore shows "Windows Server 2019 10.0 build 10240" with 8.0.6.20
and "Windows Server 2019 10.0 build 17763" with 8.0.7.10.

**Comment by pshipton** [comment]
We've got these fixes in 8.0.6.31 which I suspect resolve the detection.
https://github.com/eclipse/omr/pull/5870 "Better detection of Windows 10 in omrsysinfo_get_OS_type()"
https://github.com/eclipse/omr/pull/5873 "Compute Windows 10 build number in omrsysinfo_get_OS_version()"
